modules/developer-cli-odo-creating-a-java-microservice-jpa-application.adoc

```diff
@@ -0,0 +1,104 @@
+= Creating a Java MicroServices JPA application
+
+With `odo`, you can create and manage a sample Java MicroServices JPA application.
+
+.Procedure
+
+. Clone the sample application:
++
+[source,terminal]
+----
+$ git clone https://github.com/OpenLiberty/application-stack-samples.git
+----
+
+. Navigate to the application directory:
++
+[source,terminal]
+----
+$ cd ./application-stack-samples/jpa
+----
+
+. Initialize the project:
++
+[source,terminal]
+----
+$ odo create java-openliberty java-application
+----
+
+. Push the application to the cluster:
++
+[source,terminal]
+----
+$ odo push 
+----
++
+The application is now deployed to the cluster. 
+
+. See the status of the cluster by streaming the OpenShift logs to the terminal:
++
+[source,terminal]
+----
+$ odo log
+----
++
+Notice the failing tests because of an `UnknownDatabaseHostException` error since you application does not have a database yet:
```

Comment:
`due to` is against the IBM styleguide.
Not sure why the rest is needed to be reworded, does it sound awkward?

Reply:
Yes, it does sound awkward a bit, from the language perspective. Thanks for letting me know about the usage of because and the IBM style guide. Actually, "due to" is not against the IBM style guide. It is the word "as" and "since". Thanks a lot for this learning :) really appreciate it!!

Another try to better it, see if this helps: "Notice the test failures and `UnknownDatabaseHostException` error. This is because your application does not have a database yet:"